### DISCUSSION TITLE
LSP forum: open, online, monthly meetings. Join google group/email list

### DISCUSSION THREAD

**Discussion by evalieungh** [NorESMhub/noresm-land-sites-platform#160] [Announcements]
To anyone and everyone reading this.

Are you interested in, or planning, model experiments using the NorESM land sites platform (https://noresmhub.github.io/noresm-land-sites-platform/) or FATES? Join this open meeting series about modelling with the NorESM Land Sites Platform, CLM-FATES, local data integration, and related topics. The goal is to inspire each other, give early feedback, and learn from each other. The meeting series starts with the EMERALD project network in Norway, but is open for anyone who is interested. Spread the word to people who might want to join (but keep the zoom link away from the open web to avoid zoombombers).

To get an invitation, sign up to this google group that we use to organise the email list: [NorESM-LandSitesPlatform google group](https://groups.google.com/g/landsitesplatform)

Time: TBD, perhaps thursdays 11:15-11:45 Oslo time CET (GMT+1)
Recurrence: monthly
Place: zoom. Join google group or message Eva Lieungh to get the link
Organiser: Eva Lieungh / LSP team

Agenda will vary according to needs, but will typically include updates from attendees about what they are working on. If yo have something to present (plans, problems, preliminary results, an interesting paper you found), please say so and we can add it to the agenda.

**Comment by evalieungh** [comment]
The first LSP Forum will happen tomorrow 2023-01-19, at 11:15 [CET (UTC +1)](https://www.timeanddate.com/time/zones/cet), on zoom. Request to join the [NorESM-LandSitesPlatform google group](https://groups.google.com/g/landsitesplatform) or contact me directly to get the link. 

Rough agenda:

- intro to meeting series purpose (Eva)
- round of introductions (everyone)
- create overview of planned use cases (everyone)
- other stuff people want to bring up (everyone)

**Comment by vizh198** [comment]
hello , thank you for reaching out to me. I would like to get some more
details of this meeting and the purpose. I also further would like to get
the link for the same.

Thank you .

With regards
Vishal

**Reply by evalieungh** [reply]
Great that you are interested, Vishal! I don't want to post the link here but can you request to join the google group?

**Comment by vizh198** [comment]
Sure, I will do that. I don't have permission to join the group, i have
asked for permission

**Reply by evalieungh** [reply]
Great! Sorry for the delay, I didn't have access to the account from home. I'll approve the request now and send the link shortly.

**Comment by vizh198** [comment]
No problem, thanks! I will be joining
